aytests/dns_setup.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+set -e -x
+
+fgrep -q 'WRITE_HOSTNAME_TO_HOSTS="no"' /etc/sysconfig/network/dhcp || exit 1
```

Comment:
Yeah that is exactly what happens when doing copy paste development ;-) I'll tweak it, thanks